pipeline_steps/common.groovy

```diff
@@ -1915,6 +1919,44 @@ void createRelease(String component_text, Boolean from_rc_branch){
   )
 }
 
+void updateDependents(REPO_NAME){
+  parallelBuilds = [:]
+  allWorkflowJobs = Hudson.instance.getAllItems(WorkflowJob)
+  venv = "${WORKSPACE}/.componentvenv"
```

Comment:
I'm not sure what you mean

Reply:
Where does `"${WORKSPACE}/.componentvenv"` come from? This function assumes its existence, is that reasonable? The venv is not obviously created for shared use and there's no guarantee regarding its existence. I think it would be useful to improve the clarity of its use.